### ISSUE TITLE
What is in yout

### ISSUE THREAD

**Issue by gildas-lormeau** [luis-marcelo/freshreleases#1]

**Comment by jaapmarcus** [comment]
Mostlikely is it malware

**Comment by gildas-lormeau** [comment]
@jaapmarcus, I agree, it ticks all the boxes. However I would like to get some details from @kylemarz and @luis-marcelo.

**Comment by jaapmarcus** [comment]
It happened also on the https://github.com/mediaelement/mediaelement/wiki/Home/_compare/808f1151b09afe90757d0259dc28b7cb0f72c3c9 repo. 

Just report them and report this repo

**Comment by gildas-lormeau** [comment]
Thank you, I already reported them yesterday.
